dubbo-registry/dubbo-registry-nacos/src/main/java/org/apache/dubbo/registry/nacos/NacosRegistryFactory.java

```diff
@@ -101,7 +100,7 @@ private void setProperties(URL url, Properties properties) {
 
     private void putPropertyIfAbsent(URL url, Properties properties, String propertyName) {
         String propertyValue = url.getParameter(propertyName);
-        if (StringUtils.isNotEmpty(propertyValue)) {
+        if (propertyValue != null && propertyValue.length() != 0) {
```

Comment:
perhaps you need to `trim` first ?

Reply:
The cause of this error is that com.alibaba.nacos.client.naming.utils.StringUtils is moved to com.alibaba.nacos.client.utils.StringUtils。

by： https://github.com/alibaba/nacos/issues/1364

Reply:
> perhaps you need to `trim` first ?

the original `isNotEmpty` method doesn't `trim`, so this is ok for me